Makefile

```diff
@@ -3,11 +3,16 @@ BINARY=gatus
 # Because there's a folder called "test", we need to make the target "test" phony
 .PHONY: test
 
-install:
+.PHONY: config
+config:
+	test -f config/config.yaml || cp -r config.yaml config/config.yaml
+
+.PHONY: install
+install: config
 	go build -mod vendor -o $(BINARY) .
 
-run:
-	GATUS_CONFIG_FILE=./config.yaml ./$(BINARY)
+run: config
+	./$(BINARY)
```

Comment:
I'm not sure I understand what we're trying to fix here. The default configuration file path, if `GATUS_CONFIG_FILE` is not set, is indeed `config/config.yaml`, but the Makefile is more intended for development purposes while the default configuration file path is more intended to be used in a containerized environment.

After all, the `config` folder in Gatus' source code that you'd have in your local development environment is a package with Go code in it, so it's not really well suited to be a folder for storing the configuration file.

Reply:
@TwiN 

Default the config path is `config/config.yaml` so we just need to copy the default `config.yaml` from "./config.yaml" to the config folder. if the target config file does not exist.

> 	DefaultConfigurationFilePath = "config/config.yaml"

Reply:
> GATUS_CONFIG_FILE=./config.yaml ./$(BINARY)

We need to change the default config for development and the changes will impact the git status.

Reply:
The idea is that the Go code is made to expect being in a Docker environment, hence why the Dockerfile moves the configuration file to the config folder:
https://github.com/TwiN/gatus/blob/1aa94a33657587733571010f2495e77c515dc9c5/Dockerfile#L15

but the Makefile does not expect a containerized environment, it expects a local environment, and in that case, the config file should be in the current folder.

For development purposes, this is essentially just going to create an extra file and may add confusion when, for instance, somebody updates the wrong config file (the one in the root folder) after the one in the `config` folder has been copied by the changes you're suggesting.

Still, I think I understand what you're referring to -- if we just start the application locally (without any argument or `GATUS_CONFIG_FILE` environment variable), it will fail because it won't be able to find the file.

So instead of modifying the Makefile, I have an alternative solution.

What if we added two extra fallback paths to this function instead? https://github.com/TwiN/gatus/blob/1aa94a33657587733571010f2495e77c515dc9c5/config/config.go#L136-L145

Right now, this is the fallback order:
1. config/config.yaml
2. config/config.yml

If we added additional fallbacks in the root folder, it would look like this:
1. config/config.yaml
2. config/config.yml
3. config.yaml
4. config.yml

and we wouldn't need to update the Makefile.